filament/backend/src/metal/MetalDriver.mm

```diff
@@ -552,7 +552,7 @@
 #if defined(IOS)
     return ShaderModel::GL_ES_30;
```

Comment:
Please update this to `ShaderModel::MOBILE`.

Reply:
done